build.gradle.kts

```diff
@@ -0,0 +1,30 @@
+plugins {
+   id("ks3.conventions.base")
+   idea
+}
+
+group = "io.ks3"
+version = "0.0.1-SNAPSHOT"
+
+
+idea {
+   module {
+      isDownloadSources = true
```

Comment:
Does this force IntelliJ to fetch sources for dependencies whenever possible?

Reply:
Ummm I think basically yes. This Gradle IDEA plugin generates some sort of IntelliJ config XML, but I can't find any docs for it!

Reply:
Hehe :) Well, if it does the job.. 👍🏼